pkg/scheduler/context.go

```diff
@@ -54,6 +55,8 @@ type ClusterContext struct {
 	startTime time.Time
 
 	sync.RWMutex
+
+	healthCheckCache *dao.SchedulerHealthDAOInfo
```

Comment:
The name "lastHealthCheckResult" sounds more appropriate to me. "cache" makes me think that it's something more complicated, like a data structure or something.